webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/ElasticsearchBackupRepository.java

```diff
@@ -305,20 +307,22 @@ public List<SnapshotInfo> findSnapshots(final String repositoryName, final Long
     } catch (final Exception e) {
       if (isSnapshotMissingException(e)) {
         // no snapshot with given backupID exists
+        // Check Tasklist returns NotFoundApiException (Similar)
```

Comment:
:question: Is this a TODO?

Reply:
Just a note for me when I will unify the controllers, to double check the exceptions as they are not checked by the compiler, I'll remove it when it's merged